docs/beta/tutorial/get_version.md

```diff
@@ -0,0 +1,53 @@
+# Tutorial: Get Tamr version
+This tutorial will introduce basic usage of the Tamr Python client.  It will guide you through the process of configuring the connection to a Tamr instance and retrieving the version of that instance.
+
+## Preparation
+To complete this tutorial you will need access to a Tamr instance.  Specifically you should have a username and password that allow you to log in to Tamr, as well as the socket address of the instance.  The socket address is composed of
+1. The protocol, such as `"https"` or `"http"`
+2. The host, which may be `"localhost"` if the instance is deployed from the same machine from which your Python code will be run
+3. The port at which you access the Tamr user interface, typically `9100`
+
+When you view the Tamr user interface in a browser, the url is `<protocol>://<host>:<port>`.
```

Comment:
New GCP we create no longer shows the port address on the URL.

Reply:
Maybe we should add a sentence like "If the port is missing, the URL is simply `<protocol>://host`"

Reply:
@minroh In doing things with the Tamr client on the new GCP instance you mention does the value of `port` need to be `None`, or is it still a value like `9100` and is just not shown in the url?

Reply:
@skalish It is still 9100 but not shown in the URL.

Reply:
This is a little concerning, since I have also run into a GCP instance with the other possibility (where the url does not have the port shown and passing `port=9100` will not work but `port=None` will).  This means a user can't always tell what the `port` argument is just by knowing the url, and instead they must be familiar with the details of the deployment/network setup or figure it out through trial and error.

In this case I'm not so sure the description of the url in the tutorial is helpful.

Reply:
@skalish I did not know about the GCP instances with `port=None`. I do not know what the best description would be either. Can we get rid of the url portion if it is not going to be always useful?

Reply:
Wait, how can the port be 9100 but not shown in the URL? Is the URL being proxied to 9100? Or maybe you have some special config in your web-browser to hide ports??

AFAIK, if the port is not shown, then the port is not needed in the URL. @minroh for the case you describe, does cURL or `requests.request(...)` work when you omit the port?

Reply:
@pcattori Newer instances created with tsc do not have the port on the URL any more. For example, this is a training instance I am currently using: https://10.20.1.33/#/  If I create a tamr unify client with `port=9100` I can use APIs as usual. If I specify `port=None`, I get an `SSLCertVerificationError` error.